app/uk/gov/hmrc/eusubsidycompliance/models/json/digital/package.scala

```diff
@@ -18,50 +18,37 @@ package uk.gov.hmrc.eusubsidycompliance.models.json
 
 import cats.implicits._
 import play.api.libs.json._
-import uk.gov.hmrc.eusubsidycompliance.models.json.eis.{Params, RequestCommon}
-import uk.gov.hmrc.eusubsidycompliance.models.types.EisAmendmentType.EisAmendmentType
+import uk.gov.hmrc.eusubsidycompliance.models.json.eis.Params
 import uk.gov.hmrc.eusubsidycompliance.models.types.Sector.Sector
-import uk.gov.hmrc.eusubsidycompliance.models.types.{EORI, EisAmendmentType, IndustrySectorLimit, UndertakingName, UndertakingRef}
-import uk.gov.hmrc.eusubsidycompliance.models.{BusinessEntity, Undertaking, UndertakingBusinessEntityUpdate}
+import uk.gov.hmrc.eusubsidycompliance.models.types.{IndustrySectorLimit, UndertakingName, UndertakingRef}
+import uk.gov.hmrc.eusubsidycompliance.models.{BusinessEntity, Undertaking}
 
 import java.time.format.DateTimeFormatter
-import java.time.{LocalDate, ZonedDateTime}
+import java.time.{Clock, Instant, LocalDate, LocalDateTime, ZoneOffset, ZonedDateTime}
 
 package object digital {
 
-  private val dateFormatter: DateTimeFormatter = DateTimeFormatter.ISO_LOCAL_DATE
+  val clock: Clock = Clock.systemUTC()
```

Comment:
Do these need to be pubic?

Also, not being able to provide an alternate clock implementation could make testing problematic. I dont think this needs changing now however.

Reply:
I haven't change these . It was in package eis so I shifted those here. I can make it private

Reply:
Thanks. No worries about the clock for now.